core/src/main/java/com/flipkart/gjex/core/config/ApiService.java

```diff
@@ -38,43 +41,6 @@ public class ApiService {
     @JsonProperty("scheduledexecutor.threadpool.size")
     private int scheduledExecutorThreadPoolSize;
 
-    public int getPort() {
-        return port;
-    }
-
-    public void setPort(int port) {
-        this.port = port;
-    }
-
-    public int getAcceptors() {
-        return acceptors;
-    }
-
-    public void setAcceptors(int acceptors) {
-        this.acceptors = acceptors;
-    }
-
-    public int getSelectors() {
-        return selectors;
-    }
-
-    public void setSelectors(int selectors) {
-        this.selectors = selectors;
-    }
-
-    public int getWorkers() {
-        return workers;
-    }
-
-    public void setWorkers(int workers) {
-        this.workers = workers;
-    }
-
-    public int getScheduledExecutorThreadPoolSize() {
-        return scheduledExecutorThreadPoolSize;
-    }
-
-    public void setScheduledExecutorThreadPoolSize(int scheduledExecutorThreadPoolSize) {
-        this.scheduledExecutorThreadPoolSize = scheduledExecutorThreadPoolSize;
-    }
+    @JsonProperty("httpFilterConfig")
```

Comment:
should be named `filterConfig` since API Section itself is for HTTP

Reply:
Also, given that it contains a single boolean, may be we flatten it, and make accessLog as top level attribute.

Reply:
we should keep filterconfig as separate config for grpc and http. so it can't be top level.
in future other updates to filterconfig can come